scico/_generic_operators.py

```diff
@@ -273,12 +273,33 @@ def jvp(self, primals, tangents):
 
         return jax.jvp(self, primals, tangents)
 
+    def jhvp(self, *primals):
```

Comment:
@Michael-T-McCann: Thoughts on the name of the method? I'm open to suggestions if you think this one is not so great.

Reply:
I don't mind it given that jax has settled on `jvp` and `vjp`.